features/mbedtls/mbed/wrapper.c

```diff
@@ -0,0 +1,29 @@
+/* mbed Microcontroller Library
```

Comment:
I think we need a more descriptive filename such as `mbed_rng.c`. `wrapper.c` isn't very descriptive.

I'm also not sure this is the best place to put the file (although I have no better suggestions yet).